src/Button.jsx

```diff
@@ -0,0 +1,3 @@
+export const Button = ({ callback, text }) => {
+  return <button onClick={callback}>{text}</button>;
```

Comment:
Use this button for all cases instead of one button for each case.
In our app all buttons require a handler and a text.
Taking this in account and comment in file `App,jsx` regarding button handler, you should rewrite this component.

```suggestion
export const Button = ({ handleClick, text }) => {
  return <button onClick={handleClick}>{text}</button>;
```